### PR TITLE
chore(go): update go version for travis

### PR DESCRIPTION
**Pull request by clement-agflow**
Asana task:
https://app.asana.com/0/0/1115766138224259/f

Description:
add 1.12 version

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,4 +2,5 @@ language: go
 go:
   - 1.6
   - 1.7
+  - 1.12
```
Comment (ferhatelmas) [inline .travis.yml]: supporting last 2 versions and no tip so update accordingly.

Comment (clement-agflow) [reply .travis.yml]: done

Comment (ferhatelmas) [reply .travis.yml]: use latest of each

File: .travis.yml [hunk 2]
```diff
@@ -1,5 +1,4 @@
 language: go
 go:
-  - 1.6
-  - 1.7
-  - tip
+  - 1.11.6
+  - 1.12.1
```
Comment (ferhatelmas) [inline .travis.yml]: use `1.12.x` for automatic

Comment (clement-agflow) [reply .travis.yml]: ok

**Review by ferhatelmas — APPROVED** [review]

**Pull request merged**